k8s.gcr.io/manifests/README.md

```diff
@@ -10,9 +10,35 @@ change.
 
 Reviewers should ensure that:
 * The source registry is one that we trust.
-* There is one non-source registry for each of {us, eu, asia}.
-* The non-source registries include an appropriate "subdir" for the
+* The following destination registries must be specified:
```

Comment:
This seems like something a presubmit test should do for us to reduce review burden?